test/data/mini_example/output_res.txt

```diff
@@ -5,10 +5,9 @@
 ##INFO=<ID=END,Number=1,Type=Integer,Description="End position of SV called.",Source="iGenVarCaller",Version="1.0">
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
 chr1	57	.	N	<DEL>	9	PASS	END=70;SVLEN=-13;SVTYPE=DEL
-chr1	99	.	N	<DEL>	1	PASS	END=125;SVLEN=-26;SVTYPE=DEL
+chr1	97	.	N	<DEL>	1	PASS	END=125;SVLEN=-28;SVTYPE=DEL
```

Comment:
This comes from 94+4 right?
`read027	2048	chr1	94	60	4M46S	*	0	0	ATTTGGATCTTGACTCTGGAAAACTTTTAACGCCGGGAATCGGTAGTCCT	*	AS:i:4	NM:i:0`

Reply:
Yes, sort of. Here is the computation:
SAM and VCF are both 1-based so it's relatively easy. The first base of read027 is aligned to position 94 (that's what the POS field tells us). The alignment ends after 4 bases (CIGAR=4M46S) so that the last mapped base aligned to position 97. The first deleted base in the reference is therefore position 98. However, the VCF standard expects us to include a padding base left of the deletion. This padding base is position 97 which therefore is the POS field in the VCF.